To: vim_dev@googlegroups.com
Subject: Patch 7.3.209
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.209
Problem:    MSVC Install instructions point to wrong batch file.
Solution:   Add a batch file for use with MSVC 10.
Files:      src/msvc2010.bat, src/INSTALLpc.txt, Filelist


*** ../mercurial/vim73/src/msvc2010.bat	2011-06-12 20:35:31.000000000 +0200
--- src/msvc2010.bat	2011-06-09 05:08:35.000000000 +0200
***************
*** 0 ****
--- 1,7 ----
+ @echo off
+ rem To be used on MS-Windows for Visual C++ 2010 Express Edition
+ rem   aka Microsoft Visual Studio 10.0.
+ rem See INSTALLpc.txt for information.
+ @echo on
+ 
+ call "%VS100COMNTOOLS%%vsvars32.bat"
*** ../mercurial/vim73/src/INSTALLpc.txt	2010-08-16 21:23:30.000000000 +0200
--- src/INSTALLpc.txt	2011-06-08 06:47:46.000000000 +0200
***************
*** 28,35 ****
  9. MS-DOS
  
  The currently preferred method is using the free Visual C++ Toolkit 2008
! |msvc-2008-express|.  If you need the executable to run on Windows 98 or ME,
! use the 2003 one |msvc-2003-toolkit|.
  
  
  1. Microsoft Visual C++
--- 28,36 ----
  9. MS-DOS
  
  The currently preferred method is using the free Visual C++ Toolkit 2008
! |msvc-2008-express|, the produced binary runs on most MS-Windows systems.  If
! you need the executable to run on Windows 98 or ME, use the 2003 one
! |msvc-2003-toolkit|.
  
  
  1. Microsoft Visual C++
***************
*** 176,182 ****
      http://www.microsoft.com/express/vc/Default.aspx
  This includes the IDE and the debugger.
  
! To set the environment execute the msvc2008.bat script.  You can then build
  Vim with Make_mvc.mak.
  
  
--- 177,183 ----
      http://www.microsoft.com/express/vc/Default.aspx
  This includes the IDE and the debugger.
  
! To set the environment execute the msvc2010.bat script.  You can then build
  Vim with Make_mvc.mak.
  
  
*** ../mercurial/vim73/Filelist	2011-05-19 13:40:47.000000000 +0200
--- Filelist	2011-06-09 05:17:25.000000000 +0200
***************
*** 262,267 ****
--- 262,268 ----
  		src/bigvim.bat \
  		src/msvcsetup.bat \
  		src/msvc2008.bat \
+ 		src/msvc2010.bat \
  		src/dimm.idl \
  		src/dlldata.c \
  		src/dosinst.c \
*** ../vim-7.3.208/src/version.c	2011-06-12 20:33:30.000000000 +0200
--- src/version.c	2011-06-12 20:34:44.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     209,
  /**/

-- 
"Hit any key to continue" is a lie.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
